api/files/serializers.py

```diff
@@ -88,6 +111,7 @@ class FileSerializer(JSONAPISerializer):
         'size',
         'provider',
         'last_touched',
+        'tags',
```

Comment:
It's not enough to set `index=True` on the field because the field is a list field, and Toku doesn't like indices on list fields before there are data in the database (I suspect you saw test failures).

For special cases like this, we document the indices in scripts/indices.py.